lte/cloud/go/services/subscriberdb_cache/subscriberdb_cache/main.go

```diff
@@ -32,30 +32,34 @@ func main() {
 		glog.Fatalf("Error creating service: %+v", err)
 	}
 
+	serviceConfig := subscriberdb_cache.MustGetServiceConfig()
+	if serviceConfig.Validate() != nil {
+		glog.Fatalf("Invalid service configs for subscriberdb_cache")
```

Comment:
Include the generated error in the fatal for debuggability 

Also, move the validation inside the MustGetServiceConfig for simplicity